sci-geosciences/opencpn-plugin-weather_routing/opencpn-plugin-weather_routing-1.13.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Can we bump it to `EAPI=8` while we are touching the package anyway?

Reply:
Done